packages/plugins/graphql/package.json

```diff
@@ -70,7 +73,9 @@
     "@strapi/strapi": "4.17.0",
     "@strapi/types": "4.17.0",
     "@types/graphql-depth-limit": "1.1.5",
-    "@types/graphql-upload": "8.0.12",
+    "@types/graphql-upload": "15.0.2",
+    "@types/koa-cors": "0.0.6",
+    "@types/koa__cors": "5.0.0",
```

Comment:
Do we need both koa-cors types dependencies?

Reply:
Oops, good catch.